### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jonash871j**

### CODE REVIEW

File: src/SkySensorsAPI/ApplicationServices/WhetherStationAppService.cs [hunk 1]
```diff
@@ -1,45 +1,32 @@
...
-	}
-
 	public async Task<WeatherStationDTO> GetWeatherStation(string macAddress, long startTime, long endTime)
 	{
-		WeatherStation wsd = await wheatherStationRepository.GetWheaterStation(macAddress);
-		IEnumerable<Sensor> sensorDatas = await wheatherStationRepository.GetSensorsByMacAddress(macAddress);
-
-		List<SensorDTO> sensors = await MapSensorsAndSensorValuesToDTO(sensorDatas, startTime, endTime);
-
+		WeatherStation weatherStation = await wheatherStationRepository.GetWheaterStation(macAddress);
```
Comment (KavTV) [inline src/SkySensorsAPI/ApplicationServices/WhetherStationAppService.cs]: weatherStationRepository

File: src/SkySensorsAPI/Controllers/WhetherStationController.cs [hunk 2]
```diff
@@ -1,42 +1,37 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using SkySensorsAPI.ApplicationServices;
-using SkySensorsAPI.Models;
+using SkySensorsAPI.Models.Dto;
```
Comment (KavTV) [inline src/SkySensorsAPI/Controllers/WhetherStationController.cs]: DTO?

File: src/SkySensorsAPI/ApplicationServices/WhetherStationAppService.cs
Comment (GITPHLAP) [inline src/SkySensorsAPI/ApplicationServices/WhetherStationAppService.cs:1]: WeatherStationAppService.cs

File: src/SkySensorsAPI/ApplicationServices/WhetherStationAppService.cs [hunk 3]
```diff
@@ -1,138 +1,93 @@
-﻿using SkySensorsAPI.Models;
+﻿using SkySensorsAPI.Models.Dto;
+using SkySensorsAPI.Models.Infrastructure;
 using SkySensorsAPI.Repositories;
 using System.Net.NetworkInformation;
 
 namespace SkySensorsAPI.ApplicationServices;
 
 public interface IWheatherStationAppService
```
Comment (GITPHLAP) [inline src/SkySensorsAPI/ApplicationServices/WhetherStationAppService.cs]:
```suggestion
public interface IWeatherStationAppService
```

**Review by GITPHLAP — DISMISSED** [review]

**Review by KavTV — APPROVED** [review]

**Pull request merged**